addons/l10n_ar_withholding/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="name">account.payment.register.form</field>
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="l10n_latam_check.view_account_payment_register_form"/>
+        <field name="arch" type="xml">
+            <group>
+                <group name="withholdings" colspan="2" 
+                    invisible="country_code != 'AR'">
+                    <field name="withholding_ids" invisible="not can_edit_wizard or (can_group_payments and not group_payment)">
+                        <tree editable="bottom">
+                            <field name="withholding_sequence_id" column_invisible="True"/>
+                            <field name="tax_id" domain="[('l10n_ar_withholding_payment_type', '=', parent.partner_type), ('company_id', '=', parent.company_id)]" options="{'no_open': True, 'no_create': True}"/>
```

Comment:
Remove company domain and put check_company=True on the tax_id in the fields

Reply:
done